configure.ac

```diff
@@ -1239,6 +1240,7 @@ elif test x$with_runtime_preset = xbitcodeinterp; then
    enable_cooperative_suspend_default=yes
    # FIXME: use TEST_PROFILE=testing_aot_full_interp
    TEST_PROFILE=testing_aot_bitcode_interp
+   PLATFORM_TEST_HARNESS_EXCLUDES=BitcodeNotSupported
```

Comment:
@rolfbjarne as discussed on Slack, please remove that change as the interpreter supports exception filters so the tests will work in that configuration.

Reply:
I added that 🙂  follow-up discussion was that the BCL code contains exception filters.